ccm_web/client/src/models/canvas.ts

```diff
@@ -2,3 +2,9 @@ export interface CanvasCourseBase {
   id: number
   name: string
 }
+
+export interface CanvasCourseSection {
+  id: number
+  name: string
+  // total_students?: number
```

Comment:
why is this commented out? This should be included

Reply:
It was not used in my changes so I left it to be added later...

Reply:
Sometimes I put in code that isn't used yet and I get asked to take it out, so I thought this was a happy medium, but I'll just uncomment it.